integration/presto/README.md

```diff
@@ -113,6 +116,10 @@ Please follow the below steps to query carbondata in presto
   enable.unsafe.in.query.processing property by default is true in CarbonData system, the carbon.unsafe.working.memory.in.mb 
   property defines the limit for Unsafe Memory usage in Mega Bytes, the default value is 512 MB.
   If your tables are big you can increase the unsafe memory, or disable unsafe via setting enable.unsafe.in.query.processing=false.
+  
+  If you do not want to use unsafe memory at all please set the below properties to false as well.
```

Comment:
currently , carbon-presto integration doesn't support unsafe, so here the description should be : please disable unsafe feature for presto integration module.